tests/sessions_tests/tests.py

```diff
@@ -727,6 +727,32 @@ def response_500(requset):
         # The value wasn't saved above.
         self.assertNotIn("hello", request.session.load())
 
+    def test_session_save_on_greater_than_500(self):
+        def response_503(request):
+            response = HttpResponse("Service Unavailable")
+            response.status_code = 503
+            request.session["hello"] = "world"
+            return response
+
+        request = self.request_factory.get("/")
+        SessionMiddleware(response_503)(request)
+
+        # The value wasn't saved above.
+        self.assertNotIn("hello", request.session.load())
+
+    def test_session_save_on_less_than_500(self):
+        def response_404(request):
+            response = HttpResponse("Page not Found")
+            response.status_code = 404
+            request.session["hello"] = "world"
+            return response
+
+        request = self.request_factory.get("/")
+        SessionMiddleware(response_404)(request)
+
+        # The value was saved above.
+        self.assertIn("hello", request.session.load())
```

Comment:
This pass without the patch. Additional tests are welcome but should be moved to a separate commit.

Reply:
Got it. Will remove this test testing status code below 500.